types/json-rpc-ws/json-rpc-ws-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as JsonRpcWs from 'json-rpc-ws';
+
+let server = JsonRpcWs.createServer();
+
+server.expose('mirror', function mirror(params, reply)
```

Comment:
this code is part of the official documentation of the package, I just wanted to have it as close as possible to the official documentation. Should I remove the name anyway ?